picard/mbjson.py

```diff
@@ -131,7 +138,11 @@ def _relations_to_metadata(relations, m):
             if 'attributes' in relation:
                 attribs = [a for a in relation['attributes']]
             if reltype in ('vocal', 'instrument', 'performer'):
-                name = 'performer:' + _parse_attributes(attribs, reltype)
+                if config.setting['use_instrument_credits']:
```

Comment:
I'd cache `config.setting['use_instrument_credits']` using a variable outside the loop over `relations`